src/js/components/Anchor/StyledAnchor.js

```diff
@@ -14,7 +14,7 @@ const StyledAnchor = styled.a`
   line-height: inherit;
   color: ${props =>
     (props.theme.dark ? props.theme.global.colors.darkBackground.text
-      : props.theme.anchor.color)};
+      : props.theme.anchor.color || colorForName('brand', props.theme))};
```

Comment:
This assumes that the theme wants anchors to use the brand color vs. something like underlining. It would think themes should opt in to how they want to style anchors.

Reply:
they can right? by using anchor.color ?

Reply:
But, if they don't want brand color, they have to set a specific color. I was more thinking that changing it to `props.theme.global.colors.lightBackground.text` as the basic behavior would be more appropriate.